### DISCUSSION TITLE
How to set Rule2 or Rule3 to one-shot(once)?

### DISCUSSION THREAD

**Discussion by 49studebaker** [arendst/Tasmota#19139] [Q&A]
If I use the command:

Rule 5

The first rule, Rule1, is set to once.

How do I set Rule2 or Rule3 to once?

**Comment by pkkrusty** [comment]
Rule1 5 or Rule2 5 or Rule3 5. Using commands without a number, like Power ON or Rule 5 usually defaults to 1. Which is what you have experienced.
